Calculator/Operations.cs

```diff
@@ -47,33 +35,47 @@ public static double GetSecondNumber()
             }
             else
             {
-                WriteLine("That is an invalid number. Please try again.");
                 return GetSecondNumber();
             }
-        }  
+        }
 
-        public static double CalculateSumOfNumbers(string calculationChoice, double firstNumber, double secondNumber)
+        public static double CalculateSumOfNumbers(double firstNumber, double secondNumber)
         {
-            double sum = 0; 
-
-            if(calculationChoice == "A")
+            var calculationChoices = new Dictionary<char, string>()
             {
-                sum = AddNumbers(firstNumber, secondNumber);
-            }
-            else if (calculationChoice == "B")
-            {
-                sum = SubtractNumbers(firstNumber, secondNumber);
-            }
-            else if (calculationChoice == "C")
-            {
-                sum = MultiplyNumbers(firstNumber, secondNumber);
-            }
-            else if (calculationChoice == "D")
+                { 'A', "Addition" },
+                { 'B', "Subtraction" },
+                { 'C', "Multiplication" },
+                { 'D', "Division" }
+            };
+
+            double sum = 0;
+
+            while(sum == 0)
```

Comment:
what if first and second number are 0?

Reply:
Then the whole thing breaks :D haha, first bug look at me go